src/dbnode/client/config.go

```diff
@@ -312,8 +312,10 @@ func (c Configuration) NewAdminClient(
 			size = *c.AsyncWriteWorkerPoolSize
 		}
 
+		workerPoolInstrumentOpts := iopts.SetMetricsScope(writeRequestScope)
```

Comment:
super nit: should this maybe be a sub-scope of writeRequestScope? maybe `.Subscope("workerpool"` or something